src/material-experimental/mdc-tabs/tab-header.scss

```diff
@@ -10,8 +11,18 @@
   @include tabs-common.paginated-tab-header-item-wrapper('.mat-mdc-tab-header');
 }
 
-// For the tab element, default inset/offset values are necessary to ensure that
-// the focus indicator is sufficiently contrastive and renders appropriately.
-.mat-mdc-tab::before {
-  margin: 5px;
+.mat-mdc-tab {
+  // For the tab element, default inset/offset values are necessary to ensure that
+  // the focus indicator is sufficiently contrastive and renders appropriately.
+  &::before {
+    margin: 5px;
+  }
+
+  // When a tab is disabled in high contrast mode, set the text color to the disabled
+  // color, which is (unintuitively) named "GrayText".
+  @include cdk.high-contrast(active, off) {
+    &[aria-disabled="true"] {
+      color: GrayText;
```

Comment:
There's also a DisabledText color IIRC. Should we be using that instead?

Reply:
Nevermind, I just double-checked it and it's indeed `GreyText`. We may want to check it against Firefox though since I saw this comment: https://github.com/angular/components/blob/main/src/material/legacy-core/option/option.scss#L40